GoogleForCreators/web-stories-wp
# Links: Increase icon crop size from 32px to 96px

## Context

Increase cropped form link icon size

## Summary

When user selects an icon for the link, we used to cropped it to the size 32x32 which is blurry when we were displaying it as 54x54 icon on form. Now increased the size to 96x96 so that if in future, we want to display it even a little bigger than 54x54, won't be a problem.

## Relevant Technical Choices

N/A

## To-do

N/A

## User-facing changes

It doesn't affect any user facing functionality, but the image of icon will be displayed sharp rather than pixelated.

Before:
<img width="299" alt="Screenshot 2021-10-13 at 10 06 21 PM" src="https://user-images.githubusercontent.com/14026160/137180587-f14a7248-1782-42fd-bbcf-cd5760bd00b1.png">

After:
<img width="313" alt="Screenshot 2021-10-13 at 10 06 09 PM" src="https://user-images.githubusercontent.com/14026160/137180613-5639b272-c79d-4780-b616-e323ace89b20.png">

Difference in Image: (right one is after the changes)
<img width="328" alt="Screenshot 2021-10-13 at 10 06 26 PM" src="https://user-images.githubusercontent.com/14026160/137180641-e6b5e01d-6bb1-4878-94e1-faeb32e3a395.png">


## Testing Instructions

This PR can be tested by following these steps:

1. Go to the story editor.
2. Add link to a layer and select the icon.
3. The form link Icon should appear sharp now instead of Pixelated.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist



- [ ] This PR addresses an existing issue and I have linked this PR

---

Fixes #9396
